app/assets/stylesheets/base/_forms.scss

```diff
@@ -18,7 +18,7 @@ label {
   margin-bottom: $base-spacing / 4;
 
   &.required:after {
-    content: "*";
+    content: '*';
```

Comment:
You might have been a little zealous on the quotes. I'd say when starting a project, just take it on a file-by-file basis and don't worry about converting the whole app. You're much more likely to run into bugs this way.